components/input/input.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>input</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="input.css">
+    <link rel="stylesheet" href="../button/button.css">
+    <link rel="stylesheet" href="../common.css">
+</head>
+<body>
+    <h1>Input</h1>
+    <p>By using below mentioned css classes, you can mirror the below components.</p>
+
+    <h3>Input textbox:</h3>
+    <div class="container">
+        <!-- Input textbox -->
+        <div class="input-container">
+            <form class="input-flex">
+                <label for="name">First Name</label>
+                <input class="input" type="text" id="name" name="name">
+                <label for="email">Email Id<span class="input-required">*</span></label>
+                <input class="input" type="text" id="email" name="email" required> 
```

Comment:
For the email type inputs, set the type attribute as "email" to setup the browser level validation.

Reply:
ok